frontend/src/utilities/__tests__/useFetchState.spec.ts

```diff
@@ -108,7 +108,7 @@ describe('useFetchState', () => {
     expect(result[1][1]).toBe(true);
 
     // rerender but with a promise that doens't resolve
-    renderResult.rerender(() => new Promise(() => null), [11], {
+    renderResult.rerender(() => Promise.resolve([] as number[]), [11], {
```

Comment:
The comment clearly states `rerender but with a promise that doens't resolve`. This change now renders with a resolved promise. Please revert and address any linting issues another way to keep the integrity of the test case.

Reply:
@christianvogt Tried fixing with the following solution. Please let me know if there is a better solution.

```
    renderResult.rerender(
      () =>
        new Promise<never>(() => {
          /* no action */
        }),
      [11],
      {
        initialPromisePurity: true,
      },
    );
  ```

Reply:
Yes the above seems fine as the logic is the same. The lint rule of no empty blocks kicks in where you need a comment in the block.